content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md

```diff
@@ -48,10 +48,17 @@ Before we begin, let’s move into a directory that we can work in. For our exam
 mkdir ~/enforce-demo && cd $_
 ```
 
-To install `chainctl`, we’ll use the `curl` command to pull the application down.
```

Comment:
Does having an [install chainctl](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding/#step-1--install-chainctl) section of the onboarding still serve us if we have a separate how to install guide? There are a lot of prereqs in this doc so I don't want to weigh it down. How are we using this doc today?

Reply:
I'll keep separate for now